### PR TITLE
Fix bug in updatecheck.py and add utfcpp to its dependency list

### PR DESCRIPTION
**Pull request by defuse**

### CODE REVIEW

**Comment by mdr0id** [comment]
This might need recent packages from:
```
zcash_proofs-0.1.0.crate
zcash_primitives-0.1.0.crate
pairing-0.15.0.crate
group-0.2.0.crate
ff-0.5.0.crate
ff_derive-0.4.0.crate
bellman-0.2.0.crate
rust-std-1.36.0-x86_64-apple-darwin.tar.gz
```

The `utfcpp` was prior to this update.

File: qa/zcash/updatecheck.py [hunk 1]
```diff
@@ -354,7 +358,7 @@ def main():
 
     if len(unchecked_dependencies) > 0:
         unchecked_dependencies.sort()
-        print("WARNING: The following dependences are not being checked for updates by this script: " + unchecked_dependencies)
+        print("WARNING: The following dependences are not being checked for updates by this script: " + ', '.join(unchecked_dependencies))
```
Comment (daira) [inline qa/zcash/updatecheck.py]:
```suggestion
        print("WARNING: The following dependencies are not being checked for updates by this script: " +
              ', '.join(unchecked_dependencies))
```

Comment (defuse) [reply qa/zcash/updatecheck.py]: Thanks!

**Comment by daira** [comment]
utACK 11470a3107d34c3982695aa76740eff6beec5f36. @zkbot r+

**Pull request merged**